src/loader/endpoints/get_inloco_cities.py

```diff
@@ -1,51 +1,169 @@
 import pandas as pd
+import sys
+import numpy as np
+
+sys.path.append("..")
+sys.path.append("../src")
```

Comment:
Do you need it?

you should always run the pipeline using the root dir, so this is not needed.